Cargo.toml

```diff
@@ -46,4 +46,7 @@ tendermint-abci  = { version = "0.34", default-features = false }
 tendermint-proto = { version = "0.34", default-features = false }
 tendermint-rpc   = { version = "0.34", default-features = false }
 
+[patch.crates-io]
+ibc        = { git = "https://github.com/cosmos/ibc-rs.git", revision = "b2d426f" }
+ibc-query  = { git = "https://github.com/cosmos/ibc-rs.git", revision = "b2d426f" }
```

Comment:
`revision` was supposed to be [`rev`](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories). :smile:

Reply:
Oh, no 😄